res/enkf/enums/realization_state_enum.py

```diff
@@ -18,11 +18,11 @@
 
 class RealizationStateEnum(BaseCEnum):
     TYPE_NAME = "realisation_state_enum"
-    STATE_UNDEFINED = None
-    STATE_INITIALIZED = None
-    STATE_HAS_DATA = None
-    STATE_LOAD_FAILURE = None
-    STATE_PARENT_FAILURE = None
+    STATE_UNDEFINED = 1
+    STATE_INITIALIZED = 2
+    STATE_HAS_DATA = 4
+    STATE_LOAD_FAILURE = 8
+    STATE_PARENT_FAILURE = 16
 
 
 RealizationStateEnum.addEnum("STATE_UNDEFINED", 1)
```

Comment:
Is this still needed? Or is the init above ever needed?

Reply:
If I don't change the initialisation from `None` to ints, `mypy` complains.
If I remove the init, tests start failing, but I have not investigated why.
I'm sure this could do with a bit of work, but perhaps not in this PR?